### PR TITLE
Prepare a new release

### PR DESCRIPTION
**Pull request by mariusbarth**
@crsh: This PR is for housekeeping. I will link all PRs from the CRAN milestone that are already addressed on the development branch. Consequently, when you merge this PR, they will automatically be closed.

### CODE REVIEW

File: R/apa_print.R [hunk 1]
```diff
@@ -1,26 +1,14 @@
...
-#'  body of an R Markdown document using an in-line code chunk.
-#'
-#'  By contrast, `$table` may be used to include a complete results table
-#'  into the document with the help of [apa_table()].
-#'
-#'  - `$estimate` contains (sometimes standardized) estimates of effect sizes,
-#'  - `$statistic` contains the results from a corresponding statistical test,
-#'  - `$full_result` combines `estimate` and `statistic`, and
-#'  - `$table` contains a tabular summary of the analysis.
+#' @evalRd apa_resutls_return_value()
```
Comment (mariusbarth) [inline R/apa_print.R:11]: type `apa_results_return_value()`

File: R/apa_print_glm.R [hunk 2]
```diff
@@ -1,60 +1,54 @@
-#' Format statistics (APA 6th edition)
+#' Typeset statistics from GLM
 #'
-#' These methods take [glm][glm()] and [lm][lm()] objects to create formatted character
-#' strings to report the results in accordance with APA manuscript guidelines.
+#' These methods take (general) linear model objects to create formatted
```
Comment (mariusbarth) [inline R/apa_print_glm.R:3]: I think this should be "(generalized) linear model", I would also spell out GLM in the title

File: R/apa_print_list.R [hunk 3]
```diff
@@ -1,57 +1,53 @@
-#' Format Statistics from Linear-Model Comparisons (APA 6th edition)
+#' Typeset statistics from Linear-Model Comparisons
```
Comment (mariusbarth) [inline R/apa_print_list.R:1]: should be title case

File: R/apa_print_nlme.R [hunk 4]
```diff
@@ -1,12 +1,17 @@
-#' Format Statistics from Nonlinear Mixed-Effects Models (APA 6th edition)
+#' Typeset statistics from nonlinear mixed-effects models
```
Comment (mariusbarth) [inline R/apa_print_nlme.R:1]: title case

File: R/apa_print_wsci.R [hunk 5]
```diff
@@ -1,11 +1,13 @@
-#' Format Within-Subjects Confidence Intervals (APA 6th edition)
+#' Typeset within-subjects confidence intervals
```
Comment (mariusbarth) [inline R/apa_print_wsci.R:1]: title case

**Review by mariusbarth — COMMENTED** [review]
Added some column-name translations for t tests in R devel

**Comment by mariusbarth** [comment]
I suspect that the failing CI on Windows with R 3.6 is only noise caused by all the tidyverse fuzz going on in our CI workflow. What do you think, isn't it ime for merging?

**Comment by crsh** [comment]
I think we are almost done. There is one remaining bug on the project list. Also, could your review the changes I have made to the two vignettes?

**Comment by mariusbarth** [comment]
After some minor changes, I am fully happy with both vignettes :+1:

**Pull request merged**